SNEWS_PT/snews_pt_utils.py

```diff
@@ -266,126 +265,56 @@ def heartbeat_data(machine_time=None,
     heartbeat_dict = dict(zip_iterator)
     return heartbeat_dict
 
-# TODO: still need it ???!
-def _check_aliases(tier):
-    tier = tier.lower()
-    coincidence_aliases = ['coincidence', 'c', 'coincidencetier', 'coinc']
-    significance_aliases = ['significance', 's', 'significancetier', 'sigtier']
-    timing_aliases = ['timing', 'time', 'timingtier', 'timetier', 't']
-    false_aliases = ['false', 'falseobs', 'retraction', 'retract', 'r', 'f']
-    heartbeat_aliases = ['heartbeat', 'hb']
-
-    if tier in coincidence_aliases:
-        tier = 'CoincidenceTier'
-    elif tier in significance_aliases:
-        tier = 'SigTier'
-    elif tier in timing_aliases:
-        tier = 'TimeTier'
-    elif tier in false_aliases:
-        tier = 'FalseOBS'
-    elif tier in heartbeat_aliases:
-        tier = 'Heartbeat'
-    else:
-        click.secho(f'"{tier}" <- not a valid argument!', fg='bright_red')
-        sys.exit()
-    return [tier]
-
-
-# def _check_cli_request(requested):
-#     """ check the requested tier in the CLI
-#
-#         Parameters
-#         ----------
-#         requested : `list`
-#             The list of requested tiers
-#
-#     """
-#     from .snews_pub import SNEWSTiers
-#
-#     valid_tiers_names = ['CoincidenceTier', 'SigTier', 'TimeTier']
-#     valid_tiers = [CoincidenceTier, SignificanceTier, TimingTier]
-#     tier_pairs = dict(zip(valid_tiers_names, valid_tiers))
-#     other_tiers_names = ['Heartbeat', 'FalseOBS']
-#     other_tiers = [Heartbeat, Retraction]
-#     other_pairs = dict(zip(other_tiers_names, other_tiers))
-#
-#     tier_name_pair = []
-#     click.secho('\nRequested tiers are; ', bold=True)
-#     for i, tier in enumerate(requested):
-#         tier = tier.lower()
-#         tiername = _check_aliases(tier)[0]
-#         if tiername in valid_tiers_names:
-#             tier_name_pair.append((tier_pairs[tiername], tiername))
-#         elif tiername in other_tiers_names:
-#             click.echo(click.style(f'\t\t> {tiername}\n', fg='yellow', bold=True) +
-#                        '\t\t    has its own separate function !\n'
-#                        f'\t\t    See ' + click.style(f'{other_pairs[tiername]}', fg='yellow'))
-#             if i == len(requested): return None
-#         else:
-#             return None
-#
-#     tiers_unique, names_unique = [], []
-#     for Tier, name in tier_name_pair:
-#         if name not in names_unique:
-#             names_unique.append(name)
-#             tiers_unique.append(Tier)
-#             click.secho(f'\t\t> {name}', fg='cyan')
-#     return tiers_unique, names_unique
-
 
 def _tier_decider(data:dict) -> tuple:
     """ Decide on the tier(s) or commands (Heartbeat/Retraction)
         Based on the content of the message
 
     """
-    from .snews_pub import SNEWSTiers
```

Comment:
Wouldn't it be best to have the imports  at the top of the script ?

Reply:
This was in the previous version, in the latest commit I completely removed `SNEWSTiers` import as we don't need it.
Little more on that; I was looking at its signature and then assign the remaining keys to be `meta` but now that `SNEWSTierPublisher` doesn't take arguments (instead takes key-value pairs as kwargs) it doesn't have a clear signature, thus here I added a `valid_keys` list. Now, if the input keys are not in this list, they are assigned as `meta`.

In `SNEWSTiersPublisher` user can still see the relevant keys in docstring and also on message schema